composer.json

```diff
@@ -15,7 +15,8 @@
     },
     "require": {
         "php": "^8.0",
-        "eventsauce/clock": "^1.0"
+        "eventsauce/clock": "^1.0",
+        "eventsauce/object-hydrator": "^0.4.2"
```

Comment:
I think the `composer.json` schema limits what we can do here, but how would you feel about moving this dependency to be a `require-dev` dependency and adding it to `suggests` instead of requiring it? I know we'd non longer be able to enforce version compatibility with `eventsauce/object-hydrator` if we did that, but I'm not sure how I feel about making `eventsauce/object-hydrator` a requirement when it isn't actually required.

Otherwise, I'm :+1: on getting this in!

Reply:
I thought about this, normally I don't like having optional dependencies but rather provide an integration package that bundles the integration code with a hard dependency. Here, the additional package might not always be in use, but it will not cause a conflict for users as it has the same base dependencies. I'm not decided, but leaning towards generally including it.